FlowCrypt/src/main/java/com/flowcrypt/email/database/dao/PubKeyDao.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors: DenBond7
+ */
+
+package com.flowcrypt.email.database.dao
+
+import androidx.room.Dao
+import androidx.room.Query
+import com.flowcrypt.email.database.entity.PublicKeyEntity
+import kotlinx.coroutines.flow.Flow
+
+/**
+ * @author Denis Bondarenko
+ *         Date: 10/20/21
+ *         Time: 4:58 PM
+ *         E-mail: DenBond7@gmail.com
+ */
+@Dao
+interface PubKeyDao : BaseDao<PublicKeyEntity> {
+  @Query("SELECT * FROM public_keys")
+  suspend fun getAllPublicKeys(): List<PublicKeyEntity>
+
+  @Query("SELECT * FROM public_keys")
+  fun getAllPublicKeysFlow(): Flow<List<PublicKeyEntity>>
```

Comment:
Both of these methods have different goals.

> suspend fun getAllPublicKeys(): List<PublicKeyEntity>

This one is used to fetch pub keys once per call. For example, when I just need to know the size of a table in some coroutines methods

> fun getAllPublicKeysFlow(): Flow<List<PublicKeyEntity>>

This one is used to fetch keys and observe any changes in a table. For example, when I show a list of keys and would like to update this list when some key was removed/updated/added.

Theoretically, I can use `fun getAllPublicKeysFlow(): Flow<List<PublicKeyEntity>>` for the first case, but it will be an uncommon approach that uses additional resources.

Reply:
Could you turn `SELECT * FROM public_keys` into a constant and reuse for both? (not in this PR)

Reply:
Sure. Will do